src/hotspot/cpu/aarch64/interp_masm_aarch64.cpp

```diff
@@ -393,7 +393,13 @@ void InterpreterMacroAssembler::dispatch_base(TosState state,
                                               bool verifyoop,
                                               bool generate_poll) {
   if (VerifyActivationFrameSize) {
-    Unimplemented();
+    Label L;
+    subw(rscratch1, rfp, esp);
```

Comment:
Use 64-bit arithmetic here, or you'll miss some kinds of error.

Reply:
Okey, Thanks.

Reply:
`subw` has been replaced as `sub`